src/Grpc.Core.Api/Status.cs

```diff
@@ -88,7 +88,8 @@ public override string ToString()
     {
         if (DebugException != null)
         {
-            return $"Status(StatusCode=\"{StatusCode}\", Detail=\"{Detail}\", DebugException=\"{DebugException}\")";
+            return $"Status(StatusCode=\"{StatusCode}\", Detail=\"{Detail}\"," +
+                $" DebugException=\"{DebugException.GetType().Name}: {DebugException.Message}\")";
```

Comment:
a slightly odd choice to show the type name - especially as concrete exception types are often internal, but: not *invalid* - just me adding a "are you 100% sure you want this?" check

Reply:
Showing the type name is normal in `Exception.ToString()`